package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "webpack -d --watch",
     "build": "webpack -p",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon server.js"
+    "start": "node server.js"
```

Comment:
Random idea: Add a `start:watch` script for using `nodemon` if Heroku benefits from it?